### PR TITLE
Issue 473

### LINKED ISSUE #473: Alias bug
It seems like data model diagrams don't get created correctly when the !alias keyword is used:
Using:
```
Project:
    _:
        App
App:
    !type User:
        id <: uuid
        acc <: Account
    !type Account:
        foo <: string
    !alias uuid:
        int
```
project

## Expected behaviour:
The undefined note `_1` should be an alias type, rendering a graph like this:

project

### PR DESCRIPTION
**Pull request by ericzhang6222**
Fixes #473  .

Changes proposed in this pull request:
- Add code to process !alias for primitive data type, and add id as default property.

@anz-bank/sysl-developers

### CODE REVIEW

**Comment by Joshcarp** [comment]
Hey, generally it's a good idea to use the default PR format and give relevant titles to PRs including linking to the original issue in the description.

**Comment by Joshcarp** [comment]
With that being said do you have any code examples or diagrams of the final result?

**Comment by ericzhang6222** [comment]
> Hey, generally it's a good idea to use the default PR format and give relevant titles to PRs including linking to the original issue in the description.

 Thanks Josh, I have updated comments.

**Comment by ericzhang6222** [comment]
> With that being said do you have any code examples or diagrams of the final result?

Uploaded following final diagrams:
decimal
int
string

File: tests/data.sysl [hunk 1]
```diff
@@ -42,3 +44,12 @@ ObjectModel:
...
     customer <: Customer
+
+PrimitiveAliasModel:
+  !type User:
+        id <: uuid
+        acc <: Account
+  !type Account:
+        foo <: string
+  !alias uuid:
+        int
```
Comment (anzdaddy) [inline tests/data.sysl]: Missing \n

**Comment by ericzhang6222** [comment]
> Please update the existing test cases and the relevant golden files as well.

Has updated, can you review again?

File: tests/data.sysl (hunk 1 above)
Comment (ericzhang6222) [reply tests/data.sysl]: @anzdaddy , updated, can you review again?

**Pull request merged**